R/install.R

```diff
@@ -129,6 +129,7 @@ install <- function(packages = NULL,
   libpaths <- renv_libpaths_resolve(library)
   renv_scope_libpaths(libpaths)
 
+  is_binary <- !identical(type, "source")
```

Comment:
You could also do something like `renv_scope_envvars(RENV_PPM_ENABLED = "FALSE")` to avoid needing to pass this along the call stack, and also ensure transformation is handled consistently anywhere else we do this, e.g.

https://github.com/rstudio/renv/blob/47e0d46d489057de87f1e004d560ef7a11cdbadb/R/pak.R#L60-L62

Reply:
Or maybe new `renv_scope_ppm_repos()` which could automatically check `renv_ppm_enabled()`?

Reply:
Good idea!